internal/config/config_file.go

```diff
@@ -7,14 +7,23 @@ import (
 	"io/ioutil"
 	"os"
 	"path"
+	"runtime"
 	"syscall"
 
 	"github.com/mitchellh/go-homedir"
 	"gopkg.in/yaml.v3"
 )
 
 func ConfigDir() string {
-	dir, _ := homedir.Expand("~/.config/gh")
+	dirPath := userHomeDir()
+
+	if runtime.GOOS == "windows" {
+		dirPath += "\\AppData\\Local\\Github Cli"
```

Comment:
Protip: you can use `filepath.Join()` here

```go
filepath.Join(dirPath, "AppData", "Local", "GitHub CLI")
```

However:
1. Shouldn't we be reading from the `%AppData%` environment variable for this?
2. Should we consider using `Roaming` instead of `Local` so that the user's gh config gets synced between their computers?
3. `CLI` should be spelled uppercase.